README.md

```diff
@@ -47,13 +47,13 @@ We have provided unit tests for you to run. A complete project will pass all pro
 To run the tests, in the command line, navigate to the project root and then run:
 
 ```bash
-$ npm test test/adagrams.test.js
+$ yarn test test/adagrams.test.js
```

Comment:
I was getting a babel warning which seemed to be related to the 
```json
  ["@babel/plugin-proposal-class-properties", {
    "loose": true
  }],
```

rule in `babel.config.js`. We're having students fill in plain methods, so this shouldn't apply.

I also added a class into my code and tested calling static methods that accessed static properties, and removing this line didn't cause any warning there either, so I think it's would be safe to remove.

Reply:
I removed it and then checked the learn submission, and the removal broke the learn submission, so I'm putting it back